### ISSUE TITLE
Swipe-up queue

### ISSUE THREAD

**Issue by ggdorman** [TeamNewPipe/NewPipe#5235]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
Hi. I would like to see a swipe-up queue implemented into the NewPipe player. I created a few mockups to give a rough idea of what it should look like. It should be accessible with a simple, you guessed it, swipe up gesture. I find it to be really useful—better than accessing it from the notification.

An alternative option would be to use the space below the video player to always show the queue (and option I think would be useful to implement either way). This would behave just like the "comments" and "next and similar videos" fragments.

#### Is your feature request related to a problem? Please describe it

Accessing the playing queue from the notification is a bit cumbersome.

#### Additional context

Mockups:
P.S. Sorry if they are big, they seem to show up full-size in the preview :/
![20201220_174614](https://user-images.githubusercontent.com/76399981/102719213-b74a2b00-42e4-11eb-800f-c216d9a4dc69.jpg)
![20201220_174707](https://user-images.githubusercontent.com/76399981/102719214-b913ee80-42e4-11eb-8422-61b309058f08.jpg)

#### How will you/everyone benefit from this feature?

It will improve the UX of the app, making it easier for people to access the playing queue, especially if they don't know it can only be accessed from the notification.

**Comment by mhmdanas** [comment]
I totally agree that accessing the play queue is more cumbersome than necessary, and I like your idea, but I have one concern. When the video is minimized, will both the minimized player (the one you see at the bottom when you minimize a video while in the app) and the "minimized queue" (I think that's a good name for the feature) be shown? That would be a bit crowded.

**Comment by vkay94** [comment]
>An alternative option would be to use the space below the video player to always show the queue (and option I think would be useful to implement either way). This would behave just like the "comments" and "next and similar videos" fragments.

@ggdorman What do you think about (https://github.com/TeamNewPipe/NewPipe/pull/4534#issuecomment-742448622)?

**Comment by i5513** [comment]
I think queue should be accesible from principal menu, at left, Trends, suscriptions, news, playlists, queue, Download and history. I had to search at github to know that I could access to queue from notifications

Thank you very much for NewPipe !!


T

**Comment by mhmdanas** [comment]
@i5513 the problem with that is that you won't be able to access the queue unless you go back to the home page, which would be a shame.

**Comment by ggdorman** [comment]
@mhmdanas  I agree that that would be problematic. With regards to #4534, I think it's also a good option, but as stated in the comment it might cause issues with gestures conflicting.

**Comment by rsyqvthv** [comment]
I think a separated playlist named "queue" or "watch later" would more reliable than queue function, and the playlist has the `remove watched` command in top right 3 dots menu make it even more convenience.

the queue is quite easily to be overwrite when I share youtube link to Newpipe, and there isn't any history of the last queue.

that's makes queue function quite unuseable.

**Comment by SameenAhnaf** [comment]
I think, managing queues with swipe-up gesture needs to have a slight tweak as it is already implemented for minimized player info. Similar gestures in almost same area will confuse the users.

@mhmdanas @ggdorman What do you think about my proposal? Users in #5785 report messing up pause and close button.
Swipe up: Manage the queue
Swipe down: Close the video
Short press on video thumbnail: Open playing video info
Short press on title bar: Pause the video
No need to keep buttons. No chance of messing up things!

@rsyqvthv That's not what this issue is asking for. Your suggestion asks for managing queues and playlists later. This issue asks for managing queues right now. However, #5827 requests for auto-deletion of played videos from enqeued lists and playlists.
